### DISCUSSION TITLE
FS22

### DISCUSSION THREAD

**Discussion by iljarogoff** [Mogli12/KeyboardSteer#728] [General]
Hi, just asking here because there no other way to contact you... will there be a rework of this great addon for FS22? Even if it is "just" for GPS

**Comment by cabmac** [comment]
Hi...its not only GPS...keyboard steering, cams,front- and rearhydraulics and PTO, shifting behaviour, braking behaviour...etcetc....

Steering with keyboard is awfull without Moglis mod....

**Comment by c0nnex** [comment]
I had a quick look at the code. 
There would be quite some work to do to make it FS22 compatible. Giant changed a LOT in their apis.

I'll happily volunteer, but not enough time to do it alone

**Comment by AgroMeester** [comment]
Hi Stefan,
Another big fan here wondering if you're having plans converting this gem to fs22.
Totally agreeing with @cabmac on this one.

Greetings from Holland!

**Comment by Mogli12** [comment]
I am working on it. I think the 1st version will exclude the transmission. But I am having problems with the conversion. It will take some time.

**Comment by BizzWareX** [comment]
it would be nice, when you bring the GPS as an Standalone mod out

**Comment by AgroMeester** [comment]
Thanks for your reply Stefan. Much appreciated. Good luck with figuring out all the new stuff. We will be waiting patiently.

**Comment by Fendtfan84** [comment]
Thanks @Mogli12  for this genial mod for the FS22.
I love your mod's♥

**Comment by Sacred-Shadow** [comment]
Verdammt ja! ... Giants sollte dir für die Mod was vom Gewinn abgeben.
Nur dank VCA ist das Spiel wirklich so richtig gut spielbar!

**Comment by iljarogoff** [comment]
Vielen vielen Dank für deine Arbeit! Du hast mir sehr große Freude bereitet. Bin gerade am Testen und muss sagen, es läuft schon ziemlich gut. Falls ich auf Bugs stoßen sollte, werde ich berichten.

**Comment by Williwillswisse** [comment]
Hallo Stefan, vielen Dank für VCA im LS22. Die Community ist außer sich 😀
Der Mario hat mit mir gleich eine Vorstellung gemacht als ich VCA hier für den LS22 gefunden hatte.
https://www.youtube.com/watch?v=wGGSP3hVpNc

**Comment by RamonaTigier** [comment]
Thank you!

**Comment by BetaTGC** [comment]
Hiya, just wanted to say thank you for the amazing mod, and then follow it up by asking a question, is there anyway i can save my settings so i dont have to do them every single time i buy a new vehicle? i always have to change the steering strenght, i make it strong, then 40% on each slider, then steering auto center off. also permanent keep speed is also changed to on, i would really like to save those setting so they apply to every vehicle i buy, if there is anyway i can do this, i would really apreciate the help, i love this mod, so thank you for all the hard work and effort that you have put into it.
